vispy/visuals/colorbar.py

```diff
@@ -409,33 +557,414 @@ def border_width(self, border_width):
+#         -------
+#         clim: tuple(min, max)
+#         """
+
+#         return self._clim
+
+#     @clim.setter
+#     def clim(self, clim):
+#         self._clim = clim
+#         # new TextVisuals need to be created
+#         self._update()
+
+#     @property
+#     def label(self):
+#         """ The vispy.visuals.TextVisual associated with the label
+#         """
+#         return self._label
+
+#     @label.setter
+#     def label(self, label):
+#         self._label = label
+#         # position, styling has to be applied
+#         self._update()
+
+#     @property
+#     def ticks(self):
+#         """ The vispy.visuals.TextVisual associated with the ticks
+
+#         Returns
+#         -------
+#         ticks: [vispy.visual.TextVisual]
+#             The array is of length 2
+#         """
+#         return self._ticks
+
+#     @ticks.setter
+#     def ticks(self, ticks):
+#         self._ticks = ticks
+#         self._update()
+
+#     @property
+#     def border_width(self):
+#         """ The width of the border around the ColorBar in pixels
+#         """
+#         return self._border_width
+
+#     @border_width.setter
+#     def border_width(self, border_width):
+#         self._border_width = border_width
+#         # positions of text need to be changed accordingly
+#         self._update()
+
+#     @property
+#     def border_color(self):
+#         """ The color of the border around the ColorBar in pixels
+#         """
+#         return self._border_color
+
+#     @border_color.setter
+#     def border_color(self, border_color):
+#         self._border_color = Color(border_color)
+#         self._border_program.frag['border_color'] = self._border_color.rgba
+
+#     @staticmethod
+#     def _prepare_transforms(view):
+#         # transfrorm = view.transforms.get_transform()
+
+#         program = view.view_program
+#         border_program = view._border_program
+
+#         border_program.vert['visual_to_doc'] = \
+#             view.transforms.get_transform('visual', 'document')
+#         border_program.vert['doc_to_render'] = \
+#             view.transforms.get_transform('document', 'render')
+
+#         program.vert['transform'] = view.transforms.get_transform()
+
+#         # program.vert['transform'] = transforms.get_full_transform()
+#         if view._label is not None:
+#             view._label._prepare_transforms(view)
+
+#         if view._ticks is not None:
+#             for tick in view._ticks:
+#                 tick._prepare_transforms(view)
+
+#     def draw(self):
+#         """Draw the visual
+
+#         Parameters
+#         ----------
+#         transforms : instance of TransformSystem
+#             The transforms to use.
+#         """
+
+#         self._program.draw('triangles')
+#         self.set_gl_state(cull_face=False)
+#         self._border_program.draw("triangle_strip")
+
+#         self._label.draw()
+
+#         for tick in self._ticks:
+#             tick.draw()
```

Comment:
I think you can just delete all of these, they are in the `git` history if we ever need them